### ISSUE TITLE
Orientation change slow...

### ISSUE THREAD

**Issue by theframework** [Telerik-Verified-Plugins/WKWebView#166]
When changing orientation... the width of the document seems slow to catchup with the chnage...  any ideas/suggestions?

**Comment by EddyVerbruggen** [comment]
video?

**Comment by theframework** [comment]
Nope... No video... Just HTML pages... Using Angular.... It's like the body doesn't expand until the orientation has finished

Thanks for replying so quickly

Rich

**Comment by EddyVerbruggen** [comment]
Hey Rich,

Sorry, I was typing quickly during dinner. What I meant was: I don't think anything can be tweaked for this issue, but it would be helpful to at least see what you're seeing, so if you can record a quick video of the problem that would be awesome.

**Comment by theframework** [comment]
Hi... Thanks for getting back to me again...

Below is a download link...

http://we.tl/Js6Jqzz3PL

video 4003 is the exact app on safari
video 4004 is on cordova WKWebView iOS9

Would love to know you throughts...

Thanks for the help once again....

**Comment by EddyVerbruggen** [comment]
Excellent video's, thanks!

I'm mostly wondering where that blue background color is coming from while you're rotating the device. That may be key figuring this out.

Would it be possible for me to receive your XCode project? Stripped from sensitive bits of course, as long as it reproduces the issue.

**Comment by theframework** [comment]
The blue background is what I set the web view background Color to, to demo the issue.

I cannot supply the source unfortunately as I am under contract not to.

Any ideas?  The orientation seems not to fire/update until fully complete

Rich

**Comment by EddyVerbruggen** [comment]
It's something that needs debugging, but I'm afraid I won't get far without your app (I haven't seen this with a plain new Cordova CLI app, have you?).

Are you building locally or with a cloud service?

Can't you strip out ALL js? I'm guessing we don't need much logic to reproduce this..

**Comment by theframework** [comment]
Hi,  thanks very much for your continued support!

It's a local build.  I'll have a look at a plan app... I build geretthing locally

I'll see what can do about a 'clean' version.... How is best to send? 



Rich

**Comment by EddyVerbruggen** [comment]
That's much appreciated!

You can send it to me at eddyverbruggen at gmail dot com
